_rules/label-content-name-mismatch-2ee8b8.md

```diff
@@ -90,7 +131,29 @@ Not all of visible label is included in accessible name.
 <button name="link" aria-label="the full">The full label</button>
 ```
 
-### Inapplicable
+#### Failed example 3
+
+The full visible label is split out across the accessible name and is therefore not [included](#included-characters) in it.
+
+```html
+<head>
+  <style>
+    .sr-only {
+      position:absolute;
+      left:-10000px;
+      top:auto;
+      width:1px;
+      height:1px;
+      overflow:hidden;}
+  </style>
+</head>
+
+<body>
+  <a href="home.html">Link <span class="sr-only">that opens a new tab</span> to the homepage</a>
```

Comment:
This seems like it meets the expectation. There are two visible text nodes, `link ` and ` to the homepage`, both of which are included in the accessible name. If they need to be treated as a single test string, we need to say that.

Reply:
Hmmm, I think this is a different interpretation of "in their entirety" than I originally had, but I now see and agree that this is a problem.

I have the feeling that adding the single word "together" in the expectation would solve this, however I am unsure where this should be placed grammatically speaking.

Reply:
Good catch! The expectation should be akin to "the visible [child text content](https://dom.spec.whatwg.org/#concept-child-text-content)" instead.

Reply:
Changed the linked definition as suggested by @kasperisager, but I didn't really think that was the problem understanding wise for most of us, so I have added a note that clarifies what can also be read out of the used definitions for "included characters" and "matching characters".

@WilcoFiers and @ShadowBB, does this solve your concerns?

Reply:
@annethyme That solves it for me.